config/opensearch_dashboards.yml

```diff
@@ -29,6 +29,10 @@
 # dashboards. OpenSearch Dashboards creates a new index if the index doesn't already exist.
 #opensearchDashboards.index: ".opensearch_dashboards"
 
+# OpenSearch Dashboards uses an index in OpenSearch to store dynamic configurations.
+# This shall be a different index from opensearchDashboards.index.
+# opensearchDashboards.config_index: ".opensearch_dashboards_config"
```

Comment:
Looks like we use snake case instead of camel case which is used in all other configs for the config name/key, is it expected?

Reply:
Updated to be camel case. Thanks for the call out.